### PR TITLE
Converted test projects and samples to run on net46

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: samples/MusicStore/ForTesting/Mocks/OpenIdConnect/OpenIdConnectBackChannelHttpHandler.cs [hunk 1]
```diff
@@ -13,10 +13,12 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
             var response = new HttpResponseMessage();
 
             var basePath = Path.GetFullPath(Path.Combine(
-#if NET451
+#if NET46
                 AppDomain.CurrentDomain.BaseDirectory,
-#else
+#elif NETCOREAPP1_1
                 AppContext.BaseDirectory,
```
Comment (pranavkm) [inline samples/MusicStore/ForTesting/Mocks/OpenIdConnect/OpenIdConnectBackChannelHttpHandler.cs]: Think you might be able to remove this ifdef

**Comment by kichalla** [comment]
This PR is blocked on https://github.com/aspnet/EntityFramework/issues/8021

**Comment by smitpatel** [comment]
Blocking EF issue has been fixed.

**Comment by kichalla** [comment]
c42617e0577375a47041acd9824aed83a7bc8c56

**Pull request closed without merging**